### ISSUE TITLE
FR: Firebase Performance Web - Add config to disable global performance api hooks 

### ISSUE THREAD

**Issue by statop** [firebase/firebase-js-sdk#7536]
### Operating System

web

### Browser Version

all

### Firebase SDK Version

0.6.4

### Firebase SDK Product:

Performance

### Describe your project's tooling

Angular app with angular cli and @angular/fire

### Describe the problem

We have many usages of the web performance api to measure timings outside of firebase.

This results in quite a bit of extra data and noise getting sent to firebase.

It would be cool to have a config that turned off the automatic logging of all web performance api usages.

### Steps and code to reproduce issue

Use the web performance api for anything. Note that all it's data is sent to firebase.

We would like to disable the automatic logging of all web performance api usages.

**Comment by oscarvaldes** [comment]
👍

**Comment by jbalidiong** [comment]
Hi @statop, thanks for reaching out to us. Does our documentation: [Disable Firebase Performance Monitoring](https://firebase.google.com/docs/perf-mon/disable-sdk?platform=web) cover the feature you are asking?

**Comment by statop** [comment]
> Hi @statop, thanks for reaching out to us. Does our documentation: [Disable Firebase Performance Monitoring](https://firebase.google.com/docs/perf-mon/disable-sdk?platform=web) cover the feature you are asking?

No, that disables everything or the automatic metrics like first paint. We want those.

We just want to disable firebase performance hooking into the window.performance api. That hook is sending _all_ usages of window.performance to firebase. That generates a tremendous amount of noise. We use that api all over the place to measure other things.

**Comment by jbalidiong** [comment]
Thanks for the clarification. I'll get this routed to the right team(s) for consideration.

I'll also leave this feature request open, so our engineers and other members of the community can share their thoughts about this matter.

**Comment by johnwest80** [comment]
any update on this?  there is just too much noise, an option like this would be huge.  any chance of it being implemented?  thx
